test/Confluent.SchemaRegistry.IntegrationTests/docker-compose.yaml

```diff
@@ -0,0 +1,34 @@
+version: '3'
```

Comment:
this docker compose file brings up a zookeeper and kafka instance just for the basic auth schema registry [copy/pasted from ryan]. I need to update the integration tests for .net to use docker more generally. I'll incorporate this into that when it's finally done a different PR. Also, I chose not to bother with SSL as i think that adds very marginal benefit for the effort required.